.github/workflows/update-dependencies.yml

```diff
@@ -16,25 +16,20 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-      - name: Setup .NET 9 and 10
-        uses: actions/setup-dotnet@b2ace4b12f4cec1b96b6361ff2694ba9e931ceb4 # v3.3.1
-        with:
-          dotnet-version: |
-            9.0.x
-            10.0.x
-
       - name: Restore solution
         run: ./build.sh -restore
 
       - name: Rewrite nuget.config
-        run: dotnet new nugetconfig --force
+        run: ./dotnet.sh new nugetconfig --force
 
       - name: Install dotnet-outdated
-        run: dotnet tool install --global dotnet-outdated-tool
+        run: ./dotnet.sh tool install --global dotnet-outdated-tool
 
       - name: Update packages
         continue-on-error: true
-        run: dotnet outdated --no-restore -u --exclude Microsoft.FluentUI.AspNetCore ./Aspire.slnx || echo "Some dependencies could not be updated, but continuing workflow."
+        env:
+          DOTNET_ROOT: ${{ github.workspace }}/.dotnet
```

Comment:
Maybe add a comment here? Just in case someone was wondering why this is necessary.